concepticondata/conceptlists/Castro-2015-608.tsv

```diff
@@ -367,22 +367,22 @@ Castro-2015-608-365	365	to gaze			望（远处）	to gaze into the distance
 Castro-2015-608-366	366	to face upward			仰（仰头的状态）	to face upward
 Castro-2015-608-367	367	to smell	2124	SMELL	闻（用鼻子）	to smell
 Castro-2015-608-368	368	to speak	1623	SPEAK	说（说汉话）	to speak, to say
-Castro-2015-608-369	369	to tell			说，指责（“我说你！”）	to reprimand, to advise
+Castro-2015-608-369	369	to tell	1711	TELL	说，指责（“我说你！”）	to reprimand, to advise
 Castro-2015-608-370	370	to sing	1261	SING	唱（水歌）	to sing (trad. Sui songs)
 Castro-2015-608-371	371	to scold	299	SCOLD	骂	to scold
 Castro-2015-608-372	372	to accompany	1542	ACCOMPANY	陪	to accompany
 Castro-2015-608-373	373	to ask	1309	ASK (INQUIRE)	问	to ask
 Castro-2015-608-374	374	to chat	2903	CHAT (WITH SOMEBODY)	聊天，玩	to chat, to hang out
 Castro-2015-608-375	375	to lie	601	LIE (MISLEAD)	撒谎，假	to tell lies, false
-Castro-2015-608-376	376	to agree			同意	to agree
+Castro-2015-608-376	376	to agree	3643	AGREE	同意	to agree
 Castro-2015-608-377	377	to be willing to			愿意	to be willing (to do sth.)
 Castro-2015-608-378	378	to laugh	1355	LAUGH	笑	to laugh
 Castro-2015-608-379	379	to cry	1839	CRY	哭	to cry
 Castro-2015-608-380	380	to blow	176	BLOW (WITH MOUTH)	吹	to blow
 Castro-2015-608-381	381	to untie	352	UNTIE	解（解开鞋带 ）	to untie
 Castro-2015-608-382	382	to bark	1206	BARKING	吠	to bark
 Castro-2015-608-383	383	to graze	3058	GRAZE	放（牛）	to herd, to graze
-Castro-2015-608-384	384	to grow			栽（菜）	to grow (from seedling), to plant
+Castro-2015-608-384	384	to grow	3461	GROW PLANTS	栽（菜）	to grow (from seedling), to plant
```

Comment:
The gloss in other lists is mapped to PLANT (SOMETHING). So I would remap this one as well.